app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitDocumentsApiController.php

```diff
@@ -343,4 +344,41 @@ public function removeEventType($summit_id, $document_id, $event_type_id){
         }
     }
 
+    /**
+     * @param LaravelRequest $request
+     * @param $summit_id
+     * @param $document_id
+     * @return \Illuminate\Http\JsonResponse|mixed
+     */
+    public function addFile(LaravelRequest $request, $summit_id, $document_id){
+        return $this->processRequest(function () use ($request, $summit_id, $document_id) {
+            $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->resource_server_context)->find($summit_id);
+            if (is_null($summit)) return $this->error404();
+
+            $file = $request->file('file');
+            if (is_null($file)) {
+                return $this->error412(array('file param not set!'));
+            }
+
+            $document = $this->service->addFile2SummitDocument($summit, $document_id, $file);
+
+            return $this->updated(SerializerRegistry::getInstance()->getSerializer($document)->serialize());
+        });
+    }
+
+    /**
+     * @param $summit_id
+     * @param $document_id
+     * @return \Illuminate\Http\JsonResponse|mixed
+     */
+    public function removeFile($summit_id, $document_id){
+        return $this->processRequest(function () use ($summit_id, $document_id) {
+            $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->resource_server_context)->find($summit_id);
+            if (is_null($summit)) return $this->error404();
+
+            $document = $this->service->removeFileFromSummitDocument($summit, $document_id);
```

Comment:
document_id should be parsed to intval(document_id)

Reply:
done